.circleci/config.yml

```diff
@@ -371,11 +374,13 @@ jobs:
                     esac
                 done
 
-                # In case the commit is HEAD of master branch, also tag stable.
+                # In case the commit is HEAD of latest version branch, also tag stable.
                 if [ -n "$CIRCLE_REPOSITORY_URL" ] && [ -n "$CIRCLE_SHA1" ]; then
-                    _ref="refs/heads/master"
-                    _head="$(git ls-remote "$CIRCLE_REPOSITORY_URL" "$_ref" | awk '{ print $1 }')"
-                    if [ "$CIRCLE_SHA1" = "$_head" ]; then
+                    _stable_head="$(
+                        git ls-remote --heads "$CIRCLE_REPOSITORY_URL" "v*" |
+                        awk '/refs\/heads\/v[0-9]+\.[0-9]+$/ { LH=$1 } END { print LH }')"
```

Comment:
Won't these two lines mean that branch commits (not released/tagged) will also be considered stable?

Reply:
These `stable` check only run when building against tags so it shouldn't happen.